code/modules/crew_objectives/medical_objectives.dm

```diff
@@ -5,26 +5,12 @@
 	jobs = "chiefmedicalofficer,geneticist,medicaldoctor"
 
 /datum/objective/crew/morgue/check_completion()
+	var/list/medical_areas = list(typesof(/area/medical/cryo), typesof(/area/medical/genetics/cloning), typesof(/area/medical/exam_room),
```

Comment:
Correct me if I'm wrong but wont this just make a bunch of nested lists

Reply:
I'd like to convert it to a list like list(typesof(/area/x, /area/y,)) but I'm not sure if this works yet

Reply:
You could have two lists.
One list of the areas you want to get all the types of, which you then loop through to populate the other list which will have all of the areas you care about.

Reply:
in other places this is done by var/x = typecacheof(list(
would that work here?